addons/cppcheckdata.py

```diff
@@ -81,6 +81,15 @@ def isIntegral(self):
     def isFloat(self):
         return self.type in {'float', 'double', 'long double'}
 
+    def isEnum(self):
+        try:
+            if self.typeScope.type == "Enum":
+                return True
+            else:
+                return False
+        except:
```

Comment:
hmm.. codacy complains about this "except:" there is no exception type specified. Is this intentional or can it be fixed?

Reply:
You are right!
I have made that change.
Thanks :)